src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discont36 = 20;
+  const discont7 = 50;
+  const rentPerDay = 40 * days;
+
+  if (days >= 7) {
+    return rentPerDay - discont7;
+  } else if (days >= 3) {
+    return rentPerDay - discont36;
+  }
```

Comment:
```suggestion
  if (days >= 7) {
    return rentPerDay - discont7;
  }
  
  if (days >= 3) {
    return rentPerDay - discont36;
  }
```
Why do you need else if?
Your code will not go further after return anyway